templates/organize/form/step3_organizers.html

```diff
@@ -36,98 +37,159 @@
-                                                    <path d="M19.5,12.5 L19.5,26.5" id="Line" stroke="#F9F7F7" stroke-width="3" stroke-linecap="square"></path>
-                                                    <path d="M12.5,19.5 L26.5,19.5" id="Line" stroke="#F9F7F7" stroke-width="3" stroke-linecap="square"></path>
-                                                </g>
+                        {% endwith %}
+                    </template>
+                    <div class="add-organizer">
+                        <button id="add-form" type="button" class="icon-button">
+                            <svg width="39px" height="39px" viewBox="0 0 39 39" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                                <g id="Application" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+                                    <g id="Organizers" transform="translate(-706.000000, -866.000000)">
+                                        <g id="Card" transform="translate(147.000000, 247.000000)">
+                                            <g id="Plus-button" transform="translate(559.000000, 619.000000)">
+                                                <circle id="Oval-2" fill="#CDCBCB" cx="19.5" cy="19.5" r="19.5"></circle>
+                                                <path d="M19.5,12.5 L19.5,26.5" id="Line" stroke="#F9F7F7" stroke-width="3" stroke-linecap="square"></path>
+                                                <path d="M12.5,19.5 L26.5,19.5" id="Line" stroke="#F9F7F7" stroke-width="3" stroke-linecap="square"></path>
                                             </g>
                                         </g>
                                     </g>
-                                </svg>
-                            </button>
-                        </div>
-                    </fieldset>
-                    //-->
-
+                                </g>
+                            </svg>
+                        </button>
+                    </div>
+                    <script>
+                        var formCount = {{ wizard.form.total_form_count }};
+                        $('#add-form').click(function() {
+                            var formTemplate = document.getElementById('new-organizer-form');
+                            var inputs = formTemplate.content.querySelectorAll("input");
+                            inputs.forEach(function(element) {
+                                element.name = element.name.replace('__prefix__', formCount);
+                            })
+                            var form = document.importNode(formTemplate.content, true);
+                            $('#coorganizers').append(form);
+                            formCount++;
+                            $('#id_organizers-TOTAL_FORMS').val(formCount);
+                        });
+                    </script>
```

Comment:
Can we move javascript from inline into [this file](https://github.com/DjangoGirls/djangogirls/blob/master/static/source/js/organize.js)?